parser-typechecker/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -411,15 +413,42 @@ renameBranch = InputPattern "move.namespace"
       _ -> Left (I.help renameBranch)
     )
 
+history :: InputPattern
+history = InputPattern "log"
```

Comment:
Recommend making `"history"` be the primary command name, and maybe even eliminate the alias `"log"`.

Reply:
Did this. I killed `log` as an alias and renamed some types that started with `Log` to start with `History` instead.